GoogleForCreators/web-stories-wp
# Templates: Update Beauty Template Aspect Ratio to 9:16

## Summary
Updates Beauty template to new aspect ratio and fixes other critiques put in latest template design pass for fashion.

Designs & Critiques:
https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=772%3A540

## Relevant Technical Choices
NA

## To-do
- Still waiting for grid SVG assets to replace grid pngs on here so they can support custom colors

## User-facing changes
- Beauty template styles

## Testing Instructions
Go to explore templates and view the beauty template. Each page should now match these designs:
https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=772%3A540

---



Fixes #2476

## Screenshots

<details>
<summary>Click to expand</summary>
<img width="266" alt="Screen Shot 2020-06-18 at 3 08 58 PM" src="https://user-images.githubusercontent.com/35983235/85072425-59425c80-b176-11ea-85d7-2520999fc139.png">
<img width="266" alt="Screen Shot 2020-06-18 at 3 08 52 PM" src="https://user-images.githubusercontent.com/35983235/85072427-5ba4b680-b176-11ea-850b-4192279e26b6.png">
<img width="266" alt="Screen Shot 2020-06-18 at 3 08 42 PM" src="https://user-images.githubusercontent.com/35983235/85072432-5cd5e380-b176-11ea-9e94-c8b44094bc0a.png">
<img width="268" alt="Screen Shot 2020-06-18 at 3 08 34 PM" src="https://user-images.githubusercontent.com/35983235/85072435-5d6e7a00-b176-11ea-9798-8d5b01aaf2bd.png">
<img width="269" alt="Screen Shot 2020-06-18 at 3 08 28 PM" src="https://user-images.githubusercontent.com/35983235/85072437-5e071080-b176-11ea-98e2-8dad47a970d3.png">
<img width="264" alt="Screen Shot 2020-06-18 at 3 08 21 PM" src="https://user-images.githubusercontent.com/35983235/85072439-5e071080-b176-11ea-838d-2d48d201b8a7.png">
<img width="266" alt="Screen Shot 2020-06-18 at 3 08 15 PM" src="https://user-images.githubusercontent.com/35983235/85072440-5e9fa700-b176-11ea-9dd1-1ac88fbcccf1.png">
<img width="268" alt="Screen Shot 2020-06-18 at 3 08 08 PM" src="https://user-images.githubusercontent.com/35983235/85072442-5f383d80-b176-11ea-82ea-492fb8a41988.png">
<img width="267" alt="Screen Shot 2020-06-18 at 3 08 00 PM" src="https://user-images.githubusercontent.com/35983235/85072443-5f383d80-b176-11ea-9c05-779d75167d6b.png">
</details>
